vaadin-notification-flow-parent/vaadin-notification-flow-integration-tests/src/main/java/com/vaadin/flow/component/notification/tests/NotificationView.java

```diff
@@ -93,202 +82,6 @@ private void createNotificationWithComponents() {
         notification.setId("component-notification");
         content.setId("label-inside-notification");
         buttonInside.setId("button-inside-notification");
-        addCard("Notification with components", button);
-    }
-
-    private void addStyledNotificationContent() {
-        NativeButton button = new NativeButton(BUTTON_CAPTION);
-
-        Div content = new Div();
-        content.addClassName("my-style");
-        content.setText("This component is styled using global styles");
-
-        Notification notification = new Notification(content);
-        notification.setDuration(3000);
-
-        String styles = ".my-style { color: red; }";
```

Comment:
This test was checking that Notification can be styled from the parent scope. The initial V10 version of `vaadin-notification` component was rendering the content in shadow DOM and copying matching styles there, see https://github.com/vaadin/vaadin-notification/pull/65. This logic was removed when switching to `renderer`.